generic: replace spi Silicon Labs patch with upstream version

namiltd opened the pull request:
Move the patches to the correct backport folder

PolynomialDivision commented:
Thanks. Would be nice if you than could also replace the patches by their accepted upstream equivalent.

namiltd commented:
Done.

namiltd commented:
@dangowrt Please review and accept.

dangowrt commented:
LGTM. Generally it's preferred to generate the patches from upstream Linux git checkout using `git format-patch` without any receivers, but `From` header indicating commit hash upstream. It's just how we usually do it for accepted patches, but that habit not being documented anywhere makes me lack good arguments to ask you to do that :wink:

namiltd commented:
@dangowrt If you can, do it properly instead of this Pull Request, I have no experience with this.

Ansuel commented:
@namiltd honestly i would just squash the 2 thing

namiltd commented:
@Ansuel Done

Ansuel commented on target/linux/generic/backport-6.1/412-v6.3-02-spidev-Add-Silicon-Labs-SI3210-device-compatible.patch:
```diff
@@ -0,0 +1,53 @@
+<<<<<<<< HEAD:target/linux/generic/backport-6.1/110-v6.3-0002-spidev-Add-Silicon-Labs-SI3210-device-compatible.patch
+From: Vincent Tremblay <vincent@vtremblay.dev>
+To: unlisted-recipients:; (no To-header on input)
+Cc: vincent@vtremblay.dev, "Rob Herring" <robh+dt@kernel.org>,
+	"Krzysztof Kozlowski" <krzysztof.kozlowski+dt@linaro.org>,
+	"Mark Brown" <broonie@kernel.org>,
+	"Guenter Roeck" <linux@roeck-us.net>,
+	"Stanislav Jakubek" <stano.jakubek@gmail.com>,
+	"Marcello Sylvester Bauer" <sylv@sylv.io>,
+	"Marek Vasut" <marex@denx.de>,
+	"Samuel Holland" <samuel@sholland.org>,
+	"Conor Dooley" <conor.dooley@microchip.com>,
+	"Pali Rohár" <pali@kernel.org>,
+	devicetree@vger.kernel.org, linux-kernel@vger.kernel.org,
+	linux-spi@vger.kernel.org
+Subject: [PATCH 2/2] spidev: Add Silicon Labs SI3210 device compatible
+Date: Tue, 27 Dec 2022 09:10:08 -0500	[thread overview]
+Message-ID: <20221227141011.111410-2-vincent@vtremblay.dev> (raw)
+In-Reply-To: <20221227141011.111410-1-vincent@vtremblay.dev>
+========
+From 6c9d1fd52956c3148e847a214bae9102b1811de5 Mon Sep 17 00:00:00 2001
+From: Vincent Tremblay <vincent@vtremblay.dev>
+Date: Tue, 27 Dec 2022 09:10:08 -0500
+Subject: [PATCH] spidev: Add Silicon Labs SI3210 device compatible
+>>>>>>>> a2b96889a4fbbc66c1dcf088015e6fe5e401083b:target/linux/generic/backport-6.1/412-v6.3-02-spidev-Add-Silicon-Labs-SI3210-device-compatible.patch
```
You messed up the rebase i think.

namiltd committed:
generic: replace spi Silicon Labs patch with upstream version

Replace Silicon Labs patch with upstream version as they got merged in
kernel 6.3.

Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>
[ rework commit title and description, use better patch ]
Link: https://github.com/openwrt/openwrt/pull/15300
Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>

namiltd commented:
@Ansuel Corrected
